configurations-conception-communes/pied-page-general/pied-page-contextuelle.md

```diff
@@ -0,0 +1,186 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/site-footer-contextual.html"
+date: 2022-11-30
```

Comment:
Note: We should think of a different approach to keep the history in the frontmatter

Reply:
Maybe a "Latest changes" Jekyll component

Reply:
What history?  Latest changes would be would be dateModified.  Historical changes should be part of a CMS?